### PR TITLE
Feat: Generate final contracts

### PR DESCRIPTION
**Pull request by roxdanila**
## Context
Generate the final version of contracts (that are auto-generated via scripts)

## Test plan
Manual

## Checklist

- [ ] Rebased the base branch
- [ ] Attached corresponding Github issue
- [ ] Prefixed the name with the type of change (i.e. feat, chore, test)
- [ ] Performed a self-review of my own code
- [ ] Followed the style guidelines of this project
- [ ] Made corresponding changes to the documentation
- [ ] Didn't generate new warnings
- [ ] Didn't generate failures on existing tests
- [ ] Added tests that prove my fix is effective or that my feature works

## Review

When reviewing a PR, please indicate intention in comments using the following emojis:
* :cake: = Nice to have but not essential.
* :bulb: = Suggestion or a comment based on personal opinion
* :hammer: = I believe this should be changed.
* :thinking: = I don’t understand something, do you mind giving me more context?
* :rocket: = Feedback

### CODE REVIEW

File: contracts/modules/legacy/PricesV1.sol [hunk 1]
```diff
@@ -8,155 +8,155 @@ contract PricesV1 {
...
 
-    if (id == 53) { // 0x0000000000000000000000000000000000000026
+    if (id == 65) { // 0x0000000000000000000000000000000000000026
       return 230; // 2.300000 %
     }
 
-    if (id == 57) { // 0x0000000000000000000000000000000000000029
-      return 200; // 2.000000 %
+    if (id == 66) { // 0x0000000000000000000000000000000000000025
+      return 210; // 2.100000 %
```
Comment (danoctavian) [inline contracts/modules/legacy/PricesV1.sol:155]: isn't it strange this is below 2.6%?

File: test/fork/basic-functionality-tests.js [hunk 2]
```diff
@@ -472,14 +472,23 @@ describe('basic functionality tests', function () {
   it('Submit claim for custody cover', async function () {
     await evm.impersonate(DAI_NXM_HOLDER);
     const coverBuyer = await getSigner(DAI_NXM_HOLDER);
+
     const claimsCountBefore = await this.individualClaims.getClaimsCount();
     const assessmentCountBefore = await this.assessment.getAssessmentsCount();
 
-    const segmentId = (await this.cover.coverSegmentsCount(custodyCoverId)).sub(1);
-    const requestedAmount = parseEther('1');
     const ipfsHash = '0x68747470733a2f2f7777772e796f75747562652e636f6d2f77617463683f763d423365414d47584677316f';
```
Comment (danoctavian) [inline test/fork/basic-functionality-tests.js:479]: this looks like a TX hash but i guess it doesn't matter

File: contracts/modules/legacy/PricesV1.sol (hunk 1 above)
Comment (danoctavian) [reply contracts/modules/legacy/PricesV1.sol:155]: it's not it's a fixed price product

File: scripts/v2-migration/output/staking-pool-metadata-ipfs-hashes.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+  "0": "QmP4qU9o4WqT2TLHzieFJQpQfECy5bTcyRjikjRPs57UNa",
```
[the remark this answers is not in the document]
Comment (roxdanila) [reply scripts/v2-migration/output/staking-pool-metadata-ipfs-hashes.json:2]: we don't use this file anywhere

**Pull request merged**